src/utils/config.ts

```diff
@@ -104,6 +104,16 @@ const configParsers = {
 
 		return parsed;
 	},
+	hostname(hostname?: string) {
+		if (!hostname) {
+			return 'api.openai.com';
+		}
+
+		// eslint-disable-next-line regexp/no-unused-capturing-group
+		parseAssert('hostname', /^([a-zA-Z\d-]+\.)*[a-zA-Z\d-]+\.[a-zA-Z]+$/.test(hostname), 'Must be an hostname');
```

Comment:
is this necessary? a local host or even a remote one might not follow this `x.y.z` format, why enforce?